src/test/java/mate/academy/dao/impl/RoleDaoTest.java

```diff
@@ -0,0 +1,46 @@
+package mate.academy.dao.impl;
+
+import mate.academy.AbstractTest;
+import mate.academy.dao.RoleDao;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.model.Role;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RoleDaoTest extends AbstractTest {
+    private static RoleDao roleDao;
+
+    @BeforeEach
+    public void setUp() {
+        roleDao = new RoleDaoImpl(getSessionFactory());

```

Comment:
The only way to initialize a field in test class is using BeforeAll, but then session factory won't be initialized, because BeforeAll is called before BeforeEach, and session factory initializing is in BeforeEach method in AbstractTest, so I had to do like that. By the way, Bohdan also did like that in his video.